src/newViews/Tagging/Export/NewExport/ExportSelection/index.tsx

```diff
@@ -0,0 +1,462 @@
+import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import { _cs, isNotDefined } from '@togglecorp/fujs';
+import {
+    Button,
+    ExpandableContainer,
+    TextInput,
+} from '@the-deep/deep-ui';
+
+import {
+    SECTOR_FIRST,
+    createReportStructure,
+    getContextualWidgetsFromFramework,
+    getTextWidgetsFromFramework,
+} from '#utils/framework';
```

Comment:
@samshara , has this been written in ts? If not lets create an issue to work on it.

Reply:
@AdityaKhatri it is written in typescript